gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -262,6 +274,95 @@ public void testCreateObjectWithGenerationId() throws Exception {
     }
   }
 
+  /** Test the Statistics of http GET request */
+  @Test
+  public void testHttpRequestStat() throws IOException {
+
+    MockHttpTransport transport =
+        mockTransport(
+            jsonDataResponse(newStorageObject(BUCKET_NAME, OBJECT_NAME)),
+            jsonErrorResponse(ErrorResponses.NOT_FOUND));
+
+    GoogleCloudStorage gcs = mockedGcs(GCS_OPTIONS, transport, trackingRequestInitializer);
+    WritableByteChannel writeChannel = gcs.create(RESOURCE_ID);
+    assertThat(writeChannel.isOpen()).isTrue();
+    assertThat(trackingRequestInitializer).isNotNull();
+    assertThat(httpStatistics.get(GoogleCloudStorageStatistics.ACTION_HTTP_GET_REQUEST).longValue())
+        .isEqualTo(1L);
```

Comment:
The test does not make sense. Mocks indicate a object + error. And the call is for create, while we assert a GET call 

Could you please be explicit with tests. and comment the call that was made

Reply:
Updated